app/Core/Database.php

```diff
@@ -29,7 +29,9 @@ public static function getConnectionData () {
 	 * @return bool database status flag
 	 */
 	public static function initialize () : bool {
-		self::$database = new \mysqli ('localhost:3306', 'root', 'glitch123');
+		$config_data = parse_ini_file("../app/serverconfig.ini");
+
+		self::$database = new \mysqli ($config_data['location'].':'.$config_data['port'], $config_data['user'], $config_data['password']);
```

Comment:
Wrap these arguments so that they do not go over, or are at least attempting not to go over, the 80 character boundary.

Reply:
done, unsure on formatting